src/components/Table/styled.tsx

```diff
@@ -10,24 +11,46 @@ const ROW_HEIGHT = 56;
 
 export const Container = styled.div`
   ${boxSizing};
-  display: block;
+  display: flex;
+  flex-direction: column;
   max-width: 100%;
+  overflow: hidden;
 `;
 
 export const Scroll = styled.div`
-  display: block;
+  display: flex;
+  flex-grow: 1;
   max-width: 100%;
-  overflow-x: auto;
-  overflow-y: hidden;
+  overflow: auto;
   scroll-behavior: smooth;
 `;
 
 export const Table = styled.table`
   width: 100%;
   border-spacing: 0;
+  z-index: ${({ theme }) => theme.honeycomb.zIndexes.normal};
 `;
 
-export const Thead = styled.thead``;
+interface HeaderProps {
+  fixed: boolean;
+  background?: Property.Background;
+}
+
+export const Thead = styled.thead<HeaderProps>`
+  ${({ fixed }) =>
+    fixed &&
+    css`
+      position: sticky;
+      top: 0;
+      z-index: ${({ theme }) => theme.honeycomb.zIndexes.normal + 2};
```

Comment:
Btw should this zIndex be a constant in theme?  Or we can have the 2 as a constant in this file

Reply:
Done, we use this +2 in other files (like `zIndexes.modals + 2`) so I just made it a constant in this file.

Reply:
OK, thanks!